<?xml version="1.0"?>
<robot name="green_block">
	<link name="base_link">
		<visual>
			<geometry>
				<mesh filename="package://irsl_sim_environments/urdf/puzzle_blocks/green_block/green_block_base_link.dae" />
			</geometry>
		</visual>
		<collision>
			<origin xyz="0.000000000000e+00 0.000000000000e+00 0.000000000000e+00" rpy="0.000000000000e+00 -0.000000000000e+00 0.000000000000e+00" />
			<geometry>
				<box size="3.000000000000e-02 3.000000000000e-02 3.000000000000e-02" />
			</geometry>
		</collision>
		<collision>
			<origin xyz="3.000000000000e-02 0.000000000000e+00 3.000000000000e-02" rpy="0.000000000000e+00 -0.000000000000e+00 0.000000000000e+00" />
			<geometry>
				<box size="3.000000000000e-02 3.000000000000e-02 3.000000000000e-02" />
			</geometry>
		</collision>
		<collision>
			<origin xyz="0.000000000000e+00 0.000000000000e+00 3.000000000000e-02" rpy="0.000000000000e+00 -0.000000000000e+00 0.000000000000e+00" />
			<geometry>
				<box size="3.000000000000e-02 3.000000000000e-02 3.000000000000e-02" />
			</geometry>
		</collision>
		<collision>
			<origin xyz="0.000000000000e+00 3.000000000000e-02 3.000000000000e-02" rpy="0.000000000000e+00 -0.000000000000e+00 0.000000000000e+00" />
			<geometry>
				<box size="3.000000000000e-02 3.000000000000e-02 3.000000000000e-02" />
			</geometry>
		</collision>
		<inertial>
			<mass value="0.021599999999999998" />
			<!--origin xyz="7.500000000000e-03 7.500000000000e-03 2.250000000000e-02" / original params -->
      <origin xyz="2.500000000000e-03 2.500000000000e-03 2.250000000000e-02" /> <!-- hot fix for simulation -->
			<inertia ixx="1.0529999999999997e-05" ixy="1.2149999999999998e-06" ixz="-1.2149999999999998e-06" iyy="1.0529999999999997e-05" iyz="-1.2149999999999998e-06" izz="1.0529999999999997e-05" />
		</inertial>
	</link>
<!-- HotFix for gazebo (manually added) -->
<gazebo reference="base_link">
 <collision>
  <surface>
   <contact>
    <ode>
     <kp>500.0</kp>
     <kd>0.01</kd>
     <max_vel>0.0</max_vel>
     <min_depth>0.0004</min_depth>
    </ode>
   </contact>
   <friction>
    <ode>
     <mu>10.0</mu>
     <mu2>10.0</mu2>
    </ode>
   </friction>
  </surface>
 </collision>
</gazebo>
</robot>
